src/test/scala/com.snowplowanalytics.maxmind.iplookups/IpLookupsTest.scala

```diff
@@ -185,11 +186,11 @@ class IpLookupsTest extends Specification with Tables {
       val evalIpLookups = evalIpLookupsFromFiles(true, 0)
       val idIpLookups   = idIpLookupsFromFiles(true, 0)
       val ioExpected = IpLookupResult(
-        new UnknownHostException("not: Name or service not known").asLeft.some,
-        new UnknownHostException("not: Name or service not known").asLeft.some,
-        new UnknownHostException("not: Name or service not known").asLeft.some,
-        new UnknownHostException("not: Name or service not known").asLeft.some,
-        new UnknownHostException("not: Name or service not known").asLeft.some
+        new UnknownHostException("not: unknown error").asLeft.some,
+        new UnknownHostException("not: unknown error").asLeft.some,
+        new UnknownHostException("not: unknown error").asLeft.some,
+        new UnknownHostException("not: unknown error").asLeft.some,
+        new UnknownHostException("not: unknown error").asLeft.some
```

Comment:
I have noticed that this test fails in older versions also - so I am not sure what's the deal here.

LMK if this just hides a real problem or it is something that was skipped for some reason